index.html

```diff
@@ -172,6 +172,7 @@ <h3 class = "getintouch">
     <button class= "btn" type="submit">Get my resume</button>
   </div>
   </div>
+  <ul>
       <ul class="chev">
       <li class="p-lan">Languages </li> 
```

Comment:
[OPTIONAL] You have very nicely implemented the suggestions given by previous reviewers. But did you noticed that the sizes of texts have increased. So let me tell you a little bit about this thing:
The purpose of giving `h1`,` h2`, `h3` and so on, is not to give small or large font-sizes, it's just because of, we want to increase the browser accessibility, so that Google could understand that this is `h1` of this website of `h2` second sub-heading of the website.
But to meet the design, you can give custom `font-size` to any text.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/229287836-7dbc14b0-e30f-4f43-a0bc-03e84af46f00.png)| ![image](https://user-images.githubusercontent.com/61361037/229287846-98ff87c5-0534-4e6c-b7d3-490c6dfa7e25.png)|

Reply:
Thanks for this helpful information.I appreciate